signature-pad/src/main/java/se/warting/signaturepad/SignaturePadView.kt

```diff
@@ -17,6 +18,7 @@ import se.warting.signatureview.views.SignedListener
 @SuppressWarnings("LongParameterList")
 @Composable
 fun SignaturePadView(
+    modifier: Modifier = Modifier,
```

Comment:
This changes the default behavior and is no longer fillMaxSize(). But that might be fine?

Reply:
And with this in place we can update the sample and get rid of the surrounding Box :+1: 

SignaturePadView(
      modifier = Modifier
          .height(SIGNATURE_PAD_HEIGHT.dp)
          .fillMaxWidth()
          .border(
              width = 2.dp,
              color = Color.Red,
          )

Reply:
I think fine, we didn't need a box anymore to set the size.
I have tried with 
`SignaturePadView(modifier = Modifier.background(Color.Green))`
inside the box, SignaturePadView size is following the box size. 
without a box, the whole screen will show the SignaturePadView like using `fillMaxSize()` and `fillMaxHeight()` in the same time